### DISCUSSION TITLE
cài đặt streisand vpn trên ubuntu 20.04

### DISCUSSION THREAD

**Discussion by tienluan1991** [community/community#142821] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

giúp tôi cài đặt streisand vpn trên ubuntu 20.04

**Comment by EliasDeHondt** [comment]
**Hi @tienluan1991 !!!**

Here are the steps to install Streisand VPN on Ubuntu 20.04:

**Step 1: Update Your System**
Open a terminal and update your system packages:
```bash
sudo apt update -y && sudo apt upgrade -y
```

**Step 2: Install Dependencies**
Streisand requires certain dependencies to run. Install them by running:
```bash
sudo apt install git python3-pip python3-venv ansible -y
```

**Step 3: Clone the Streisand Repository**
Download Streisand from GitHub:
```bash
git clone https://github.com/StreisandEffect/streisand.git
```
Navigate to the Streisand directory:
```bash
cd streisand
```

**Step 4: Start the Installation**
Run the Streisand setup script:
```bash
./streisand
```

**Step 5: Follow the On-Screen Instructions**
During the installation, Streisand will guide you through setting up the VPN. You can choose to deploy it on your local machine or set it up on a cloud provider like AWS or DigitalOcean.

> **Note:** Streisand is a community project, and some parts of the installation may require troubleshooting or adjustments, especially if the project has had compatibility issues with Ubuntu 20.04.

If you found this response helpful, please feel free to mark it as the correct answer. 😊

**Comment by samus-aran** [comment]
Hi @tienluan1991 , thanks for participating!

We take our [[Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md)](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
